system/vmware-monitoring/values.yaml

```
@@ -0,0 +1,260 @@
+global:
+  domain: cloud.sap
+  clusterType: controlplane
+  alerts:
+    enabled: true
+    prometheus: vmware
+
+  aggregations:
+    enabled: true
+    prometheus: vmware
+
+owner-info:
+  support-group: observability
+  maintainers: 
+    - Richard Tief
+    - Tommy Sauer
+  helm-chart-url: https://github.com/sapcc/helm-charts/tree/master/system/vmware-monitoring
+
+thanos:
+  vmware: true
+  deployWholeThanos: true
+
+prometheus-server:
+  vmware: true
+
+  retentionTime: 4h
+
+  additionalScrapeConfigs: {}
+
+  scrapeInterval: "60s"
+
+  alertmanagers:
+    hosts:
+      - alertmanager-internal.scaleout.eu-de-1.cloud.sap
+      - alertmanager-internal.scaleout.eu-nl-1.cloud.sap
+
+  ingress:
+    enabled: true
+    authentication:
+      sso:
+        enabled: true
+        authTLSSecret: kube-system/ingress-cacrt
+        authTLSVerifyDepth: 3
+        authTLSVerifyClient: on
+
+  internalIngress:
+    enabled: false
+
+  persistence:
+    enabled: true
+    accessMode: ReadWriteOnce
+    size: 100Gi
```

```suggestion
    size: 10Gi
```

This is the metal variant, let's see where we go here. If we go to scaleout, we need to adjust anyway.